pkg/limayaml/defaults.go

```diff
@@ -297,7 +297,11 @@ func FillDefault(y, d, o *LimaYAML, filePath string) {
 	mounts := make([]Mount, 0, len(d.Mounts)+len(y.Mounts)+len(o.Mounts))
 	location := make(map[string]int)
 	for _, mount := range append(append(d.Mounts, y.Mounts...), o.Mounts...) {
+		if mount.SSHFS.FollowSymlinks == nil {
+			mount.SSHFS.FollowSymlinks = pointer.Bool(false)
+		}
 		if i, ok := location[mount.Location]; ok {
+			mounts[i].SSHFS.FollowSymlinks = mount.SSHFS.FollowSymlinks
```

Comment:
You only overwrite if the pointer is not `nil`:

```suggestion
			if mount.SSHFS.FollowSymlinks != nil {
				mounts[i].SSHFS.FollowSymlinks = mount.SSHFS.FollowSymlinks
			}
```
```